tests/test_db.py

```diff
@@ -138,3 +139,83 @@ def teardown_module(module):
 
         assert await db.allocate_parking_lot("test_user", 1) is True
         assert await db.allocate_parking_lot("test_user", 2) is False
+
+
+@pytest.mark.asyncio
+async def test_get_available_parking_lots(event_loop):
+    with Postgresql() as postgresql:
+        db = await DbAccess.create(postgresql.url(), event_loop, reset_tables=True)
+        parking_lot1 = ParkingLot(100, 'test_name1', 0.0, Location(0.0, 1.0089))
+        parking_lot2 = ParkingLot(100, 'test_name2', 0.0, Location(0.0, 1.01))
+        parking_lot3 = ParkingLot(100, 'test_name2', 0.0, Location(0.0, 1.0))
+
+        await db.insert_parking_lot(parking_lot1)
+        await db.update_parking_lot_availability(1, 50)
+        await db.insert_parking_lot(parking_lot2)
+        await db.update_parking_lot_availability(2, 50)
+        await db.insert_parking_lot(parking_lot3)
+        await db.update_parking_lot_availability(3, 50)
+
+        records = await db.get_available_parking_lots(lat=0, long=1, dist_meters=1000, exclusions=[])
+
+        assert len(records) == 2
+        assert records[0]['id'] == 3
+        assert records[0]['distance'] == 0
+        assert records[1]['id'] == 1
+        assert round(records[1]['distance']) == 991
+
+        records2 = await db.get_available_parking_lots(lat=0, long=1, dist_meters=1000, exclusions=[3])
+
+        assert len(records2) == 1
+        assert records2[0]['id'] == 1
+        assert round(records2[0]['distance']) == 991
+
+
+@pytest.mark.asyncio
+async def test_add_allocation(event_loop):
+    with Postgresql() as postgresql:
+        db = await DbAccess.create(postgresql.url(), event_loop, reset_tables=True)
+        parking_lot = ParkingLot(100, 'test_name', 0.0, Location(0.0, 1.0089))
+        user_id = str(uuid4())
+
+        park_id = await db.insert_parking_lot(parking_lot)
+        result = await db.allocate_parking_lot(user_id, park_id)
+        assert result
+
+        allocations = await db.get_parking_lot_allocations(park_id)
+        assert allocations[0]['park_id'] == park_id
+        assert allocations[0]['user_id'] == user_id
+
+
+@pytest.mark.asyncio
+async def test_delete_allocation(event_loop):
+    with Postgresql() as postgresql:
+        db = await DbAccess.create(postgresql.url(), event_loop, reset_tables=True)
+        parking_lot = ParkingLot(100, 'test_name', 0.0, Location(0.0, 1.0089))
+        user_id = str(uuid4())
+
+        park_id = await db.insert_parking_lot(parking_lot)
+        result = await db.allocate_parking_lot(user_id, park_id)
+        assert result
```

Comment:
Some of your assertions seem to be irrelevant to the name of your test. To me this suggests they should be in their own tests. You've already tested this above, so it seems pointless to put it here as well.

Reply:
True, I used lots of asserts to make debugging easier if the test fails in the future.